Moved retriever.php into /bin folder and various other changes.

Sweepr opened the pull request:
Moved retriever.php into /bin folder.
Changed permission denied warning when run from web browser.
Updated .htaccess in bin folder.
Updated SpotTemplateHelper.php so we can run Retrieve.php from bin folder.

Sweepr commented:
@mesa57 , Please review and merge i you agree?

mesa57 commented:
Sorry, I do NOT agree. 
Many people have current crontab entries to point at retrieve.php
We had this change before and undid it.

mesa57 commented:
Also I am not sure if this will break things : 
`IfModule mod_authz_host.c`

Sweepr commented:
> 
> 
> Sorry, I do NOT agree.
> Many people have current crontab entries to point at retrieve.php
> We had this change before and undid it.

I remember there was such change some time ago and reverted it, the reason was that it wasn't properly tested by the contributor and introduced problems.

I also believe it isn't that difficult to add /bin in the current path, just a matter of updating the path in the cron.

> Also I am not sure if this will break things :
> IfModule mod_authz_host.c

I have tested the new .htaccess with Apache 2.4.41 and it works fine.

Are you willing to give it a try? if you still feel the same then simply undo the change.

mesa57 commented:
But please first revert the move of retrieve.php
________________________________
From: Sweepr <notifications@github.com>
Sent: Saturday, April 18, 2020 5:10:17 PM
To: spotweb/spotweb <spotweb@noreply.github.com>
Cc: mesa57 <mesa57@hotmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [spotweb/spotweb] Moved retriever.php into /bin folder and various other changes. (#524)


Sorry, I do NOT agree.
Many people have current crontab entries to point at retrieve.php
We had this change before and undid it.

I remember there was such change some time ago and reverted it, the reason was that it wasn't properly tested by the contributor and introduced problems.

I also believe it isn't that difficult to add /bin in the current path, just a matter of updating the path in the cron.

Also I am not sure if this will break things :
IfModule mod_authz_host.c

I have tested the new .htaccess with Apache 2.4.41 and it works fine.

Are you willing to give it a try? if you still feel the same then simply undo the change.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/spotweb/spotweb/pull/524#issuecomment-615887026>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABQONBKPVRPWZN3RECLGADTRNG7FTANCNFSM4MLLM3DA>.

Sweepr commented:
> 
> But please first revert the move of retrieve.php
> […](#)
> ________________________________ From: Sweepr <notifications@github.com> Sent: Saturday, April 18, 2020 5:10:17 PM To: spotweb/spotweb <spotweb@noreply.github.com> Cc: mesa57 <mesa57@hotmail.com>; Mention <mention@noreply.github.com> Subject: Re: [spotweb/spotweb] Moved retriever.php into /bin folder and various other changes. (#524) Sorry, I do NOT agree. Many people have current crontab entries to point at retrieve.php We had this change before and undid it. I remember there was such change some time ago and reverted it, the reason was that it wasn't properly tested by the contributor and introduced problems. I also believe it isn't that difficult to add /bin in the current path, just a matter of updating the path in the cron. Also I am not sure if this will break things : IfModule mod_authz_host.c I have tested the new .htaccess with Apache 2.4.41 and it works fine. Are you willing to give it a try? if you still feel the same then simply undo the change. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<[#524 (comment)](https://github.com/spotweb/spotweb/pull/524#issuecomment-615887026)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABQONBKPVRPWZN3RECLGADTRNG7FTANCNFSM4MLLM3DA>.



> 
> But please first revert the move of retrieve.php
> […](#)
> ________________________________ From: Sweepr <notifications@github.com> Sent: Saturday, April 18, 2020 5:10:17 PM To: spotweb/spotweb <spotweb@noreply.github.com> Cc: mesa57 <mesa57@hotmail.com>; Mention <mention@noreply.github.com> Subject: Re: [spotweb/spotweb] Moved retriever.php into /bin folder and various other changes. (#524) Sorry, I do NOT agree. Many people have current crontab entries to point at retrieve.php We had this change before and undid it. I remember there was such change some time ago and reverted it, the reason was that it wasn't properly tested by the contributor and introduced problems. I also believe it isn't that difficult to add /bin in the current path, just a matter of updating the path in the cron. Also I am not sure if this will break things : IfModule mod_authz_host.c I have tested the new .htaccess with Apache 2.4.41 and it works fine. Are you willing to give it a try? if you still feel the same then simply undo the change. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<[#524 (comment)](https://github.com/spotweb/spotweb/pull/524#issuecomment-615887026)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABQONBKPVRPWZN3RECLGADTRNG7FTANCNFSM4MLLM3DA>.

Sure, but if i revert the move of retrieve.php all other changes (.htaccess etc.) are of no use, so i'll just cancel the pull-request all together then.

mesa57 commented:
Sorry for the inconvenience 😞
